tests/models/blip_2/test_modeling_blip_2.py

```diff
@@ -871,15 +1413,24 @@ def test_inference_opt(self):
         inputs = processor(images=image, text=prompt, return_tensors="pt").to(torch_device, dtype=torch.float16)
 
         # max_length for BLIP includes prompt length from now on, use max_new_tokens
-        predictions = model.generate(**inputs, max_new_tokens=11)
+        predictions = model.generate(
+            **inputs,
+            max_new_tokens=11,
+            do_sample=False,
+            num_beams=5,
+            top_p=0.9,
+            repetition_penalty=1.0,
+            length_penalty=1.0,
+            temperature=1,
+        )
         generated_text = processor.batch_decode(predictions, skip_special_tokens=True)[0].strip()
 
         # Test output
         self.assertEqual(
             predictions[0].tolist(),
-            [2, 24, 18, 45, 10, 343, 6, 24, 18, 10, 4105, 50118],
+            [2, 15610, 1597, 2977, 6, 13011, 1594, 43052, 50118],
         )
-        self.assertEqual(generated_text, "it's not a city, it's a beach")
+        self.assertEqual(generated_text, "san diego, california")
```

Comment:
Hmmmm - this doesn't seem right (the picture is indeed of a beach, not a city). 

Could you try the following to see if you're able to recover the previous generations: 
* Try without the additional generation kwargs
* Try without the added scaling included in the modeling file c.f. https://github.com/huggingface/transformers/pull/29261/files#r1501123761

Reply:
ok, I will change the test, I was only trying to get a similar result to `test_inference_t5` since in that one the answer shows _san diego_. it was not related to the scaling change.